dist/site.webmanifest

```diff
@@ -0,0 +1,8 @@
+{
+    "icons": [{
+        "src": "icon.png",
+        "sizes": "192x192",
+        "type": "image/png"
```

Comment:
> You don't need this.

@marcoscaceres Did some quick tests in Chrome / Chrome Beta for Android, and you are right. Commit updated! :)